doc/api/module.md

```diff
@@ -509,7 +514,7 @@ changes:
   * `specifier` {string}
   * `context` {Object}
 * Returns: {Object|Promise}
-  * `format` {string|null|undefined} A hint to the load hook (it might be
+  * `format` {string|undefined} A hint to the load hook (it might be
```

Comment:
This is documenting the `resolve` hook, not `defaultResolve`. `null` is still a valid value there